dashboards/MySQL_Query_Response_Time.json

```diff
@@ -44,6 +45,7 @@
             "type": "dashboards"
         }
     ],
+    "refresh": "1m",
```

Comment:
@askomorokhov ok, thanks for sharing, I wasn't aware we had a style guide! :)  Does this mean that @azhebel should re-submit his PR? Or what is our next step now to move this PR forward?